api_views/users.py

```diff
@@ -206,3 +206,64 @@ def delete_user(username):
                 return Response(error_message_helper("User not found!"), 404, mimetype="application/json")
         else:
             return Response(error_message_helper("Only Admins may delete users!"), 401, mimetype="application/json")
+
+
+def get_user_by_email(email):
```

Comment:
New function get_user_by_email() appears to lack unit tests. Please add corresponding test cases to verify functionality.